api/api.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Is it intentional to have a solution file here as well as `api.test/api.test.sln`?  Maybe we should move this .sln to the root since it references the api and api.test projects?

Reply:
It was not intentional, it's just that I am not very familiar with the .NET ecosystem, and solution files in particular. Both files were created by the Rider when I opened the api and api.test projects. It seems redudant to have two files indeed, especially as the api.sln contains references to both itself and api.test. I moved the api.sln to the root of the project,  changed the file references and removed api.test.sln. Thanks for your input.